packages/api-clients/open-api/bffApi.yml

```diff
@@ -4108,6 +4108,125 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/Problem"
+  /templates/eservices/{eServiceId}:
+    put:
```

Comment:
This should be a post

Reply:
The reason we are ditching the put verb in the new APIs is explained by Alessio [here](https://pagopaspa.slack.com/archives/C048JGDMJHH/p1737107041277379?thread_ts=1737106388.168579&cid=C048JGDMJHH)